website/content/docs/upgrading/upgrade-to-1.12.x.mdx

```diff
@@ -50,6 +53,12 @@ Key                                   Value
 ...
 plugin_name                           snowflake-database-plugin
 plugin_version                        v0.6.0+builtin
```

Comment:
Does `plugin_version` just show as blank if it wasn't specified in the mount call?

Reply:
Yup, specifically `n/a`